spacetemplate/assets/scrum.yaml

```diff
@@ -0,0 +1,242 @@
+space_template:
+  id: "629c963a-174c-4c37-b487-272067e88bd4"
+  name: Scrum
+  description: >
+    Scrum is an agile framework for managing work with 
+    an emphasis on software development.
+work_item_types:
+- id: &commonWorkItemTypeID "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Common Work Item Type
+  description: TBD
+  fields:
+    "system.history": 
+      label: History
+      description: The history of changes of the workitem
+      type:
+        kind: history
+      required: no
+    "system.reason": 
+      label: Reason
+      description: TBD
+      type:
+        kind: string
+      required: no
+    "system.stackrank": 
+      label: StackRank
+      description: TBD
+      type:
+        kind: float
+      required: yes
+- id: &impedimentID "35837274-b833-4001-bee9-4d69e7f8aa0c"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  name: Impediment
+  description: TBD
+  icon: fa fa-tasks
+- id: &issueID "22a6f261-d982-427a-a9c6-939ea398992b"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  name: Issue
+  description: TBD
+  icon: fa fa-tasks
+- id: &taskID "b8b7272e-5e1e-433d-965a-87c3e21dbed7"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  name: Task
+  description: TBD
+  icon: fa fa-tasks
+  child_types:
+  - *impedimentID
+  - *issueID
+  fields:
+    "system.state": 
+      label: State
+      description: The state of the work item
+      required: yes
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        # First state in this ordered map is implicitly the default state
+        values: 
+        - todo
+        - in progress
+        - done
+        - removed
+    "system.blocked": 
+      label: Blocked
+      description: TBD
+      type:
+        kind: bool
+      required: no
+    "system.remainingwork": 
+      label: RemainingWork
+      description: TBD
```

Comment:
A good description could involve the unit in which the remaining work is measured.